### ISSUE TITLE
Can't load WA anymore

### ISSUE THREAD

**Issue by WALuckyIssue** [WeakAuras/WeakAuras2#2693]
Hey everybody,

I cannot load weak auras anymore.
I have stopped playing one month after release of BFA and thought about coming back.
So I started WOW and obviously had a lot of addon errors.
I did see my weak auras were turned on, but several others caused issues (elvUI etc), so I went step by step and removing addons that don't fully work anymore.

But now weakauras isn't working anymore at all, even with weakaura only activated.
I know this screenshot is taken with elvUI turned on, but it doesn't work either way.

Help would really be appreciated. Thank you!

![errorWA](https://user-images.githubusercontent.com/74175378/98514854-6ae1e900-226a-11eb-83e3-70d977fbb045.PNG)

**Comment by mrbuds** [comment]
It seems to be the same problem as https://github.com/WeakAuras/WeakAuras2/issues/2652 which was closed by other, but we don't know how it was fixed

**Comment by WALuckyIssue** [comment]
I fixed it, by deleting all files in the SavedVariables folder.
I guess it was caused by an old aura, that doesn't work with the new version anymore.

Follow up question:
Can I restore my auras from the old files and check them seperately to see if it was only caused by a single aura?
I don't want to remake all of my auras, because that took quite a lot of work, and after my break I'm not too familiar with weakauras atm.
